plugins/providers/docker/driver.rb

```diff
@@ -42,6 +52,13 @@ def build(dir, **opts, &block)
         matches[:id]
       end
 
+      # Check if podman emulating docker CLI is enabled.
+      #
+      # @return [Bool]
+      def podman?
+        execute('docker', '-v').include?("podman")
```

Comment:
Older versions of podman don't support  the `-v` flag. Eg. if you just run a `sudo yum -y install podman` from centos without adding any more current repositories will leave you with `podman` 1.4.4 which does not have the `-v` option. However they all seem to have the `--version` option. 
```suggestion
        execute('docker', '--version').include?("podman")
```

Reply:
I added this in the newest push, thanks!